python/cuml/cuml/accel/pytest_plugin.py

```diff
@@ -14,6 +14,13 @@
 # limitations under the License.
 #
 
+from collections import defaultdict
+from importlib.metadata import version
+from pathlib import Path
+
+import yaml
```

Comment:
We should probably add `pyyaml` as a dependency to the `test_python` section in `dependencies.yaml`.